### DISCUSSION TITLE
Reveal just 2~3 characters of each secret

### DISCUSSION THREAD

**Discussion by cmygray** [community/community#25830] [Actions]
<p>Currently it reveals *** for every secrets. It’s hard to check correctness.</p>
<p>as is:</p>
<p>AWS_ACCESS_KEY_ID: ***</p>
<p>to be:</p>
<p>AWS_ACCESS_KEY_ID: a2f****</p>

**Comment by ethomson** [comment, marked as answer]
<p>Thanks for the feedback.  This is an interesting suggestion - I’m not sure if this is necessarily the implementation that we’d like to use, but I understand that we may need to provide some aide in debugging secrets.  We’ll give this some thought.  Thanks!</p>
